src/net.h

```diff
@@ -378,63 +379,66 @@ class CNode
     int nRecvVersion;
 
     // BU connection de-prioritization
-    //! Total bytes sent and received
+    //* Total bytes sent and received
     uint64_t nActivityBytes;
```

Comment:
-//* Total bytes sent and received
+//** Total bytes sent and received */

Reply:
Thx!  I reviewed the doxygen docs and //* only works for 2 line comments or more.  Weird

Reply:
But let's work thru these as part of a larger commit re-enabling doxygen for bitcoind, since @sickpig approved this PR.